pkg/network/tracer/connection/fentry/tracer.go

```diff
@@ -13,25 +13,27 @@ import (
 	"os"
 	"syscall"
 
+	manager "github.com/DataDog/ebpf-manager"
```

Comment:
Shouldn't this be with the imports below?

Reply:
nvm

Reply:
I have mine configured to separate local packages from third-party